src/types.ts

```diff
@@ -22,6 +26,7 @@ export type EntityConfig = {
 
 export type EntityConfigInternal = EntityConfig & {
   // children: string[];
+  isRemaining?: boolean;
```

Comment:
this should be removed. Probably a merge left over

Reply:
Done.